unity/Assets/Scripts/AgentManager.cs

```diff
@@ -213,7 +213,22 @@ public void Initialize(ServerAction action)
             {
                 //set up physics controller
                 SetUpArmController(true);
+
+				if(action.useMassThreshold)
+				{
+					if(action.massThreshold > 0.0)
+					SetUpMassThreshold(action.massThreshold);
+
+					else
+					{
+						var error = "massThreshold must have nonzero value if useMassThreshold = True";
```

Comment:
I was getting something strange when testing this error check. I get an in-editor debug log error of:

"ActionFinished called with agentState not in procesing"

this is in `actionFinished` inside of the `BaseFPSAgentController.cs`

It seems this only throws an editor log error, but doesn't seem to change the actual behavior of actionFinished?